here-naksha-lib-ext-manager/src/main/java/com/here/naksha/lib/extmanager/ExtensionCache.java

```diff
@@ -112,41 +112,70 @@ private void publishIntoCache(KVPair<Extension, File> result, ExtensionConfig ex
       if (!isNullOrEmpty(extension.getInitClassName())) {
         try {
           Class<?> clz = loader.loadClass(extension.getInitClassName());
-          clz.getConstructor(INaksha.class, Extension.class).newInstance(naksha, extension);
-        } catch (ClassNotFoundException
-            | InvocationTargetException
-            | InstantiationException
-            | NoSuchMethodException
-            | IllegalAccessException e) {
+          Object obj = clz.getConstructor(INaksha.class, Extension.class).newInstance(naksha, extension);
```

Comment:
As we now have init(naksha, extension) method, we should only use empty constructor for instantiating object of InitClassName

Reply:
Done.